.circleci/soltest.sh

```diff
@@ -54,7 +54,7 @@ get_logfile_basename() {
 }
 
 BOOST_TEST_ARGS="--color_output=no --show_progress=yes --logger=JUNIT,error,test_results/`get_logfile_basename`.xml"
-SOLTEST_ARGS="--evm-version=$EVM --evmonepath /usr/lib/libevmone.so $flags"
+SOLTEST_ARGS="${SOLTEST_ARGS} --evm-version=$EVM --evmonepath /usr/lib/libevmone.so $flags"
```

Comment:
Is the `$flags` argument for additional options to be passed to soltest? I was wondering if you could assign `--no-smt` to `$flags` and if that would work.

Reply:
Right, could also do that

Reply:
I'll try that in the other PR and then update this one accordingly